### ISSUE TITLE
Trim trailing commas doesn't work

### ISSUE THREAD

**Issue by bwieckow** [prantlf/jsonlint#22]
Example json:
```
{
     "ErrorInstance": {
        "type": "string",
        "description": "Description",
        "example": "/path/12345/msgs/abc"
      },
      "ErrorTitle": {
        "type": "string",
        "description": "Description",
        "example": "You do not have enough credit."
      },
      "ErrorDetail": {
        "type": "string",
        "description": "Description",
        "example": "Your current balance is 30, but that costs 50."
      },
}
```

Command:
`jsonlint --trim-trailing-commas test.json`

Output:
```
File: test.json
Parse error on line 17, column 1:
...costs 50."      },}
---------------------^
Unexpected token }
```

**Comment by prantlf** [comment]
TL;DR: Add the command-line parameter `--trailing-commas`. (Or `-T`.)

This one is tricky - you want to omit any trailing commas from the JSON output. But you get a failure from parsing the JSON input. So, before you can ask for trimming the command from the output, you have to ask for them to be accepted in the input. I admit that it could be improved by assuming `--trailing-commas` if `--trim-trailing-commas` was requested. I'll think about it.

**Comment by prantlf** [comment]
I'm going to assume `--trailing-commas` if just `--trim-trailing-commas` is requested in the upcoming release.

**Comment by prantlf** [comment]
Released with [14.1.0](https://github.com/prantlf/jsonlint/releases/tag/v14.1.0), committed in b3fb10d3a9ad0f4e6f38395fe3ccff83611d69be.
